docker-build.sh

```diff
@@ -76,14 +76,14 @@ buildOpenJDKViaDocker()
   # TODO This could be extracted overridden by the user if we support more
   # architectures going forwards
   local container_architecture="x86_64/ubuntu"
-  local build_variant_flag=""
   BUILD_CONFIG[DOCKER_FILE_PATH]="docker/${BUILD_CONFIG[OPENJDK_CORE_VERSION]}/$container_architecture"
   
   if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "openj9" ]; then
-    build_variant_flag="--openj9"
+    docker/dockerfile-generator.sh --version "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" --path "${BUILD_CONFIG[DOCKER_FILE_PATH]}" --openj9
+  else
+    docker/dockerfile-generator.sh --version "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" --path "${BUILD_CONFIG[DOCKER_FILE_PATH]}"
```

Comment:
Looks like you could just append the --openj9 in an if else block as opposed to repeating the whole line?

Reply:
Hmm, I think I may have a way of doing it as it was before (i.e. using `build_variant_flag` ), which would stop the repetition. I did change it as the `dockerfile-generation.sh` script was complaining about having a blank argument passed to it (see the hotspot builds here: https://ci.adoptopenjdk.net/job/DockerfileCheck/125/ ), however I think I've found a solution to this :-)